docs/src/concepts/kernels.md

```diff
@@ -0,0 +1,93 @@
+# Kernels
+
+The [KernelAbstactions.jl](https://github.com/JuliaGPU/KernelAbstractions.jl) package provides a macro-based dialect that hides the intricacies of vendor-specific GPU programming. It allows one to write hardware-agnostic kernels that can be instantiated and launched for different device backends without modifying the high-level code nor sacrificing performance.
+
+Followingly, we show how to write and launch kernels on various backends. We also explain the concept of a `Launcher` in [Chmy.jl](https://github.com/PTsolvers/Chmy.jl), that complements the default kernel launching, allowing us to perform more fine-grained operations on different `Field` objects.
+
+## Writing Kernels
+
+This section highlights some important features of [KernelAbstactions.jl](https://github.com/JuliaGPU/KernelAbstractions.jl) that are essential for understanding the high-level abstraction of the kernel concept that is used throughout our package. As it barely serves for illustrative purposes, for more specific examples, please refer to their [documentation](https://juliagpu.github.io/KernelAbstractions.jl/stable/).
```

Comment:
I know that KA should provide such doc, but maybe it could be beneficial to show a very simple 2D or 3D kernel in different flavours, using `NTuple`, `Cartesian`, etc... ?

Reply:
Good idea, added a table [ref](https://github.com/JuliaGPU/KernelAbstractions.jl/issues/217) for explaining indices and an example with tuple splatting.

Reply:
@luraess you can review the relevant commit [here](https://github.com/PTsolvers/Chmy.jl/pull/24/commits/5f252bb730a097b012bef9c5c781b4eda880659e).

Reply:
I guess we can keep that info yes.